assets/js/components/DashboardMainApp.js

```diff
@@ -262,12 +261,6 @@ export default function DashboardMainApp() {
 				<HelpMenu />
 			</Header>
 
-			{ ! viewOnlyDashboard && (
-				<Fragment>
-					<ConsentModeSetupCTAWidget />
-				</Fragment>
-			) }
-
```

Comment:
😍

Reply:
Yup - I had the same amount of pleasure in getting rid of this finally! And we've done the same with `ErrorNotifications` as well.